src/main/java/echoServer/Echoer.java

```diff
@@ -34,15 +38,18 @@ public boolean readClientInput(Socket clientSocket) throws IOException {
         String message;
 
         while((message = bufferedReader.readLine()) != null) {
-            return interpretClientMessage(message, printer);
+            StatusCode statusCode = StatusCode.PAGE_NOT_FOUND;
+            HTTPResponse response = new HTTPResponse(statusCode);
+            String responseText = response.buildResponse(statusCode);
+            return interpretClientMessage(responseText, printer);
         }
         return false;
     }
 
     private boolean interpretClientMessage(String message, ClientWriteable printer) throws IOException {
         if (!message.equals("byebye")) {
-            System.out.println("Echo: "+message);
-            printer.println("Echo: "+message);
+            System.out.println(message);
```

Comment:
Should an HTTP server accept a message of "byebye" from a client socket?

Reply:
Oh, right, it wouldn't. I guess every request would be formatted as a separate curl input, right?